### ISSUE TITLE
Training method of YOLO-RD

### ISSUE THREAD

**Issue by Ting-Yun-Huang** [MultimediaTechLab/YOLO#193]
Hi, thanks for releasing the awsome project!

I am interested in the training process of the newest RD module (DConv).
Since I do not find information about training this module in the repo currently, I would like to know that:

1. Are the initialized dictionary atoms treated as kernel weights of this convolution layer?
`self.D = Conv(atoms, in_channels, 1, activation=False)`

2. How do you initialize the D layer in the DConv?
According to the paper, I guess it is like `conv.weight = nn.Parameter(new_weight)`,
while the new weight coming from the cluster centroids after k-means cluatering. But I am not sure about it since there is no related initialize function implemented.

3. Are the atoms treated as learnable parameters during training? 
According to the psudo-code in the YOLO-RD paper, it seems like you update R and D simultaniously without anything freezed:
 // StandardTraining Method
 backbone,head←newModel()
 for epoche=1 to num_epochs do
     for each(img,box) ∈ Dataset do
         output←head(RD(backbone(img)))
         loss←loss_function(output,box)
         update(loss,(backbone,RD,head))
As I know, dictionary leraning usually do not train the dictionary and other parts in the model simultaniously. In the 1st step, the dictionary is freezed to get the coefficients. In the 2nd step, only the dictionary is updated while other parts are freezed. Do I need to do this during training (for example, freezing D at the 1st step training and freezing R and Backbone in the 2nd step training)? Could you further explain the training process?
